web/src/main/java/org/bkvm/cache/MetadataCache.java

```diff
@@ -146,15 +146,15 @@ public void deleteCluster(int clusterId) {
         try (EntityManagerWrapper emw = getEntityManager()) {
             EntityManager em = emw.em;
             if (clusterId == null) {
-                Query q = em.createQuery("SELECT l "
-                        + "FROM bookie l "
-                        + "ORDER BY l.bookieId", Bookie.class);
+                Query q = em.createQuery("SELECT b "
+                        + "FROM bookie b "
+                        + "ORDER BY b.clusterId", Bookie.class);

```

Comment:
probably it  better to order by clusterId AND bookieId, otherwise you will have unpredictable results

Reply:
Done